slatedb/Cargo.toml

```diff
@@ -42,16 +42,17 @@ serde = { workspace = true, features = ["derive"] }
 serde_json = { workspace = true }
 siphasher = { workspace = true }
 snap = { workspace = true, optional = true }
+thread_local = { workspace = true }
 thiserror = { workspace = true }
 tokio = { workspace = true, features = ["fs", "macros", "sync", "rt", "rt-multi-thread", "signal"] }
+tokio-util = { workspace = true, default-features = false, features = ["rt"] }
+tokio-retry2 = { version = "0.5.7", features = ["jitter"] }
```

Comment:
Haven't done any research, but did you evaluate @Xuanwo's http://github.com/Xuanwo/backon ? Not sure about the tradeoffs between these two libs.